src/assisted_test_infra/test_infra/helper_classes/day2_cluster.py

```diff
@@ -0,0 +1,285 @@
+import ipaddress
+import json
+import os
+import subprocess
+import time
+import uuid
+from abc import ABC
+
+import waiting
+
+import consts
+from assisted_test_infra.test_infra import utils
+from assisted_test_infra.test_infra.controllers.node_controllers.libvirt_controller import LibvirtController
+from assisted_test_infra.test_infra.tools import static_network
+from assisted_test_infra.test_infra.utils.waiting import wait_till_all_hosts_are_in_status
+from service_client import log
+from service_client.assisted_service_api import InventoryClient
+from tests.config import ClusterConfig, TerraformConfig, global_variables
+from tests.config.global_configs import Day2ClusterConfig
```

Comment:
Do not import tests package into test-infra package.
You can use `BaseClusterConfig` and `BaseNodeConfig` instead.
`global_variables` can be used only outside test_infra package